config/autosave.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Enable autosave
+    |--------------------------------------------------------------------------
+    |
+    | THIS IS A EXPERIMENTAL FEATURE. THINGS WILL PROBABLY GO WRONG.
+    | Set to true to enable autosave in general. You need to enable
+    | autosave manually in every collection, to make it work.
+    |
+    | For example inside `content/collections/pages.yaml`
+    | Add `autosave: true` or `autosave: 5000` for a
+    | saving interval of every 5 seconds.
+    |
+    */
+
+    'enabled' => env('STATAMIC_AUTOSAVE_ENABLED', false),
+
+    /*
+    |--------------------------------------------------------------------------
+    | Do you understand the risks?
+    |--------------------------------------------------------------------------
+    |
+    | As this is experimental and might break, be aware that you souldn't use
+    | this feature in production. If you do or if anything breaks, this is
+    | expected and will probably happen.
+    |
+    | Set `understood` to true, to confirm that you do understand the risks.
+    |
+    */
+
+    'understood' => false,
```

Comment:
I think the double opt in is a bit of overkill. 🙂 The experimental warning is already on the `enabled` setting.

Reply:
I agree :) [76bf371](https://github.com/statamic/cms/pull/6198/commits/76bf3710e800b4574f1930c15e4202c03d9c2325)